$NetBSD: patch-aa,v 1.3 2009/01/08 16:14:16 wiz Exp $

  - make variables overridable and thus the build process more flexible
    Submitted upstream.

--- Makefile.orig	2008-11-29 14:49:38.000000000 +0100
+++ Makefile
@@ -1,17 +1,21 @@
-DESTDIR=
-PREFIX=/usr/local
-MANDIR=$(PREFIX)/share/man
+DESTDIR?=
+PREFIX?=/usr/local
+MANDIR?=$(PREFIX)/share/man
 TAR=tar
 FIND=find
 SED=sed
+INSTALL_MAN_DIR?=mkdir -p
+INSTALL_SCRIPT_DIR?=mkdir -p
+INSTALL_MAN?=cp
+INSTALL_SCRIPT?=install -m 755
 
 all: manpage
 
 install: all
-	mkdir -p $(DESTDIR)$(MANDIR)/man1/
-	mkdir -p $(DESTDIR)$(PREFIX)/bin/
-	cp convmv.1.gz $(DESTDIR)$(MANDIR)/man1/
-	install -m 755 convmv $(DESTDIR)$(PREFIX)/bin/
+	${INSTALL_MAN_DIR} $(DESTDIR)$(MANDIR)/man1/
+	${INSTALL_SCRIPT_DIR} $(DESTDIR)$(PREFIX)/bin/
+	${INSTALL_MAN} convmv.1.gz $(DESTDIR)$(MANDIR)/man1/
+	${INSTALL_SCRIPT} convmv $(DESTDIR)$(PREFIX)/bin/
 
 manpage:
 	pod2man --section 1 --center=" " convmv | gzip > convmv.1.gz
